### DISCUSSION TITLE
copilot 自动扣费

### DISCUSSION THREAD

**Discussion by Arealm** [community/community#54795] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

copilot 自动扣费，明明设置了每个月最大安全支出为0美金，但是还是自动扣费了，取消订阅的入口太难找，
系统本次扣费是2023年5月8日，请帮我申请退款

**Comment by Akash1134** [comment, marked as answer]
Hi @Arealm ,

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by Arealm** [reply]
thank you
